apps/condo/pages/ticket/index.tsx

```diff
@@ -195,6 +198,17 @@ export const TicketsPageContent = ({
                                                                         {WarrantiesLabel}
                                                                     </Checkbox>
                                                                 </Col>
+                                                                <Col>
+                                                                    <Checkbox
+                                                                        onChange={handleReturnedChange}
+                                                                        checked={returned}
+                                                                        style={CHECKBOX_STYLE}
+                                                                        eventName={'TicketFilterCheckboxWarranty'}
```

Comment:
`eventName` prop is used for sending analytics data. Please use unique identifier